README.md

```diff
@@ -178,6 +188,11 @@ be passed using `[]`, for example: `:token-name[pretty]` would pass the string
 Normally formats are defined using `morgan.format(name, format)`, but for certain
 advanced uses, this compile function is directly available.
 
+The function returned takes arguments `tokens` , `req`, and `res` where `tokens`
+refers to `morgan` itself. If a log should be skipped the function will return
+`null`. The function returned or a custom function can be passed directly to 
```

Comment:
Ah. So maybe I'm just confused here. I was under impression from the issue that spawned this PR that the goal here was to document what the `format` argument in `morgan(format, options)` accepts. Are we missing that documentation and this is the documentation for `morgan.compile`? Since I mistook this as the documentation for the `format` argument, my thought here was that documentation for the `format` argument should be up in the section regarding the `format` argument, rather than in the `morgan.compile` section.